hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/BaseSparkCommitActionExecutor.java

```diff
@@ -292,6 +294,9 @@ protected void setCommitMetadata(HoodieWriteMetadata<HoodieData<WriteStatus>> re
   @Override
   protected void commit(HoodieWriteMetadata<HoodieData<WriteStatus>> result) {
     context.setJobStatus(this.getClass().getSimpleName(), "Commit write status collect: " + config.getTableName());
+    if (!config.getBoolean(IGNORE_ERROR_WRITE) && result.getWriteStatuses().filter(status -> status.getErrors().size() > 0).count() > 0) {
+      throw new HoodieException("Error writing record in the job");
+    }
     commit(result.getWriteStatuses(), result, result.getWriteStats().isPresent()
```

Comment:
should fix this method instead.

Reply:
fix it with: ignore error writestaus by filter them in default，then continue commit logic

Reply:
ok fix in BaseCommitActionExecutor::commit Method @danny0405

Reply:
@danny0405 PATL